app-emulation/retroarch/retroarch-1.7.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic
+
+DESCRIPTION="A sophisticated libretro frontend for emulators, game engines and media players"
+HOMEPAGE="https://retroarch.com/"
+SRC_URI="https://github.com/libretro/RetroArch/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+IUSE="alsa cg +opengl +pulseaudio sdl X"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+LICENSE="GPL-3"
+SLOT="0"
+
+DEPEND="
+	dev-libs/libxml2
```

Comment:
Libxml2/freetype (and maybe others) are slotted, don't we need to rebuild when the slot changes on upgrades?